src/types/shared.ts

```diff
@@ -162,21 +162,20 @@ export type FlightsConditions = {
  * passengers if you require this level of granularity.
  */
 export type OfferSliceConditions = FlightsConditions & {
-  // TODO: ask James about this
   /**
    * Whether passengers are able to select a seat prior to check in.
    */
-  advance_seat_selection: boolean
+  advance_seat_selection: boolean | null
 
   /**
    * Whether passengers are given preferential boarding over others passengers in their cabin.
    */
-  priority_boarding: boolean
+  priority_boarding: boolean | null
 
   /**
    * Whether passengers are given access to a fast track lane during check in.
    */
-  priority_check_in: boolean
+  priority_check_in: boolean | null
```

Comment:
https://duffel.com/docs/api/v2/offers#offers-schema-slices-slices-conditions-slices-conditions-priority-check-in ✅